kafka/protocol/types.py

```diff
@@ -12,8 +12,13 @@ def _pack(f, value):
         raise ValueError(error)
 
 
-def _unpack(f, data):
+def _unpack(f, data, length=None):
     try:
+        if type(data) is not str:
```

Comment:
What about`if not isinstance(data, str)` rather than limiting yourself with `type()`?

Reply:
Yep. I'll change that. Thanks!